hscontrol/db/node.go

```diff
@@ -77,6 +77,28 @@ func ListNodes(tx *gorm.DB) (types.Nodes, error) {
 	return nodes, nil
 }
 
+func (hsdb *HSDatabase) ListNodesSubset(nodeIDs types.NodeIDs) (types.Nodes, error) {
```

Comment:
```suggestion
func (hsdb *HSDatabase) ListNodes(nodeIDs ...types.NodeID) (types.Nodes, error) {
```

I would prefer to just have the same function for this, but use the list expansion argument like this so it remains optional for getting all.

Reply:
Is implemented.